src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const SUM = days * DAILY_RATE;
```

Comment:
```suggestion
  const currentPrice = days * DAILY_RATE;
```
Calculated values should be in camelCase